cli/tests/integration/compile_tests.rs

```diff
@@ -202,6 +208,38 @@ fn standalone_load_datauri() {
   assert_eq!(output.stdout, b"Hello Deno!\n");
 }
 
+#[test]
+fn standalone_load_dynamic_import() {
+  let dir = TempDir::new().expect("tempdir fail");
+  let exe = if cfg!(windows) {
```

Comment:
@williamtetlow Can we revert this change? eszip doesn't include dynamic imports in the graph as of 0.16.0 The loader should handle this instead and I'd prefer to do this in a follow up PR instead.